package.json

```diff
@@ -75,5 +86,17 @@
 		"printWidth": 120,
 		"trailingComma": "none",
 		"singleQuote": false
-	}
+	},
+	"description": "The front-end for the new StuyActivities site.",
+	"main": "index.js",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/stuysu/stuyactivities.org.git"
+	},
+	"author": "",
+	"license": "ISC",
```

Comment:
@TheEgghead27 Did we ever choose a license properly for this? I don't think we did

Reply:
Ah that's the default npm template, should be able to remove that when I get home 👍